inc/class-wpseo-replace-vars.php

```diff
@@ -1210,52 +1210,59 @@ private function add_custom_field_prefix( $field ) {
 	 * Set/translate the help texts for the WPSEO standard basic variables.
 	 */
 	private static function set_basic_help_texts() {
-		self::$help_texts['basic'] = array(
-			'date'                 => __( 'Replaced with the date of the post/page', 'wordpress-seo' ),
-			'title'                => __( 'Replaced with the title of the post/page', 'wordpress-seo' ),
-			'parent_title'         => __( 'Replaced with the title of the parent page of the current page', 'wordpress-seo' ),
-			'archive_title'        => __( 'Replaced with the normal title for an archive generated by WordPress', 'wordpress-seo' ),
-			'sitename'             => __( 'The site\'s name', 'wordpress-seo' ),
-			'sitedesc'             => __( 'The site\'s tag line / description', 'wordpress-seo' ),
-			'excerpt'              => __( 'Replaced with the post/page excerpt (or auto-generated if it does not exist)', 'wordpress-seo' ),
-			'excerpt_only'         => __( 'Replaced with the post/page excerpt (without auto-generation)', 'wordpress-seo' ),
-			'tag'                  => __( 'Replaced with the current tag/tags', 'wordpress-seo' ),
-			'category'             => __( 'Replaced with the post categories (comma separated)', 'wordpress-seo' ),
-			'primary_category'     => __( 'Replaced with the primary category of the post/page', 'wordpress-seo' ),
-			'category_description' => __( 'Replaced with the category description', 'wordpress-seo' ),
-			'tag_description'      => __( 'Replaced with the tag description', 'wordpress-seo' ),
-			'term_description'     => __( 'Replaced with the term description', 'wordpress-seo' ),
-			'term_title'           => __( 'Replaced with the term name', 'wordpress-seo' ),
-			'searchphrase'         => __( 'Replaced with the current search phrase', 'wordpress-seo' ),
-			'sep'                  => sprintf(
-				/* translators: %s: wp_title() function. */
+		$replacement_variables = array(
```

Comment:
there are two spaces after the comma in label argument of `WPSEO_Replacement_Variable`